package-overrides/npm/angular-dragdrop@1.0.5.json

```diff
@@ -0,0 +1,11 @@
+{
+  "registry": "jspm",
```

Comment:
We have a strange convention with the angular libraries where we typically use github repos over npm installs. The main reason for this is that npm installs do a kind of "conversion" of node packages based on the Node package resolution algorithm and environment needs. This can cause unnecessary deps to be downloaded as well.

Perhaps use the GitHub repo instead here? The override should be much the same.